src/main/java/aws/retrospective/service/SectionService.java

```diff
@@ -226,4 +202,88 @@ private User getAssignUser(AssignUserRequestDto request) {
         return userRepository.findById(request.getUserId()).orElseThrow(
             () -> new NoSuchElementException("Not Found User Id : " + request.getUserId()));
     }
+
+
+    private void validateTeamRetrospectiveAccess(GetSectionsRequestDto request, Retrospective findRetrospective) {
+        if (request.getTeamId() != null) {
+            Team findTeam = getTeam(request.getTeamId());
+            if (!findRetrospective.isSameTeam(findTeam)) {
+                throw new ForbiddenAccessException("다른 팀의 회고보드에 접근할 수 없습니다.");
+            }
+        }
+    }
```

Comment:
```suggestion
    private void validateTeamRetrospectiveAccess(GetSectionsRequestDto request, Retrospective findRetrospective) {
        if (request.getTeamId() == null) {
        	return;
        }
        Team findTeam = getTeam(request.getTeamId());
		if (!findRetrospective.isSameTeam(findTeam)) {
			throw new ForbiddenAccessException("다른 팀의 회고보드에 접근할 수 없습니다.");
		}
    }
```
early return 패턴을 사용하면 indent depth가 깊어지지 않고 가독성이 좋아지는 편!
이 메서드는 길이가 짧아서 체감을 못할 수 있는데 메서드가 커지면 큰 도움이 될 수도 있고 그렇슴다..
다른 메서드도 쭉 수정해보면 좋을듯~

Reply:
구조를 단순하게 할 수 있는 장점이 있네요!
적용해 보았습니다 ㅎㅎ